src/validations/rules/ssp.sch

```diff
@@ -4162,11 +4202,11 @@
             doc:context="oscal:information-type"
             id="information-type-has-availability-impact-diagnostic">A FedRAMP SSP information-type lacks a availability-impact.</sch:diagnostic>
         <sch:diagnostic
-            doc:assertion="cia-impact-matches-security-sensitivity-level"
+            doc:assertion="cia-impact-matches-security-objective"
             doc:context="oscal:information-type"
-            id="cia-impact-matches-security-sensitivity-level-diagnostic">The FedRAMP SSP information type -<xsl:value-of
-                select="substring-before(local-name(), '-')" /> impact- base or selected elements do not match the security sensitivity
-            level.</sch:diagnostic>
+            id="cia-impact-matches-security-objective-diagnostic">The FedRAMP SSP security objective <xsl:value-of
+                select="substring-before(local-name(), '-')" /> does not match the <xsl:value-of
+                select="substring-before(local-name(), '-')" /> impact value.</sch:diagnostic>
```

Comment:
When the related assertion fails, the message `The FedRAMP SSP security objective security does not match the security impact value.` is the result.

Reply:
This has yet to be corrected.

When the related assertion fails, the diagnostic message "The FedRAMP SSP security objective **security** does not match the **security** impact value." is the result.

"security objective security" does not seem appropriate. Neither does "security impact".

The context of the assertion is `oscal:security-objective-confidentiality | oscal:security-objective-integrity | oscal:security-objective-availability`. Perhaps you should use the third tuple of the context element names, something like `tokenize(local-name(),'-')[3]`.

Also the `doc:context` attribute value does not match the assertion context.

The following lines are currently located at 4243-4248.
```
        <sch:diagnostic
            doc:assertion="cia-impact-matches-security-objective"
            doc:context="oscal:information-type"
            id="cia-impact-matches-security-objective-diagnostic">The FedRAMP SSP security objective <xsl:value-of
                select="substring-before(local-name(), '-')" /> does not match the <xsl:value-of
                select="substring-before(local-name(), '-')" /> impact value.</sch:diagnostic>
```

Reply:
No idea what happened there as I had the change in previously.  Obviously I didn't get it into the commit.  

The correct value is now outputting.

Description: An SSP security objective value must be the same as highest available value of the same information type impact element. 
The FedRAMP SSP security objective **confidentiality** does not match the **confidentiality** impact value.